metadata-comparison/src/main/java/org/codice/nitf/metadatacomparison/FileComparer.java

```diff
@@ -549,8 +554,8 @@ private void outputRPCs() throws IOException {
         }
         if (!rpc.keySet().isEmpty()) {
             out.write("RPC Metadata:\n");
-            for (String tagname : rpc.keySet()) {
-                out.write(String.format("  %s=%s\n", tagname, rpc.get(tagname)));
+            for (Map.Entry<String,String> entry : rpc.entrySet()) {
+                out.write(String.format("  %s=%s%n", entry.getKey(), entry.getValue()));
```

Comment:
Not sure how that space got deleted above when this one wasn't disturbed 🤔 At any rate, I fixed it.